packages/ag-charts-website/src/content/docs/localisation/index.mdoc

```diff
@@ -0,0 +1,304 @@
+
+Please see [All Built-In Translations](./localisation/#all-built-in-translations) for a full list of built-in translations.
+{% /warning %}
+
+### Framework Integration Examples
+
+{% tabs %}
+
+{% if isFramework("javascript", "angular", "vue") %}
+
+{% tabItem id="FormatJS" label="FormatJS" %}
+
+```js
+import { createIntl } from '@formatjs/intl';
+
+const messages = {
+    'context-menu.download': 'Téléchargez une copie de ce tableau',
+};
+
+const intl = createIntl({
+    locale: 'fr-FR',
+    messages: messages,
+});
+
+AgCharts.create({
+    // ...
+    locale: {
+        messages: messages,
+        formatMessage({ id, params }) {
+            return intl.formatMessage({ id }, params);
+        },
+    },
+});
+```
+
+{% /tabItem %}
+
+{% tabItem id="I18Next" label="I18Next" %}
+
+```js
+import i18next from 'i18next';
+
+const messages = {
+    'context-menu.download': 'Téléchargez une copie de ce tableau',
+};
+
+await i18next.init({
+    lng: 'fr',
+    resources: {
+        fr: {
+            translation: messages,
+        },
+    },
+});
+
+AgCharts.create({
+    // ...
+    locale: {
+        messages: messages,
+        formatMessage({ id, params }) {
+            return i18next.t(id, params);
+        },
+    },
+});
+```
+
+{% /tabItem %}
+
+{% /if %}
+
+{% if isFramework("react") %}
+
+{% tabItem id="React Intl" label="React Intl" %}
+
+```jsx
+import { IntlProvider, useIntl } from 'react-intl';
+
+const messages = {
+    'context-menu.download': 'Téléchargez une copie de ce tableau',
+};
+
+const MyChart = () => {
+    const intl = useIntl();
+
+    const options = {
+        // ...
+        locale: {
+            messages: messages,
+            formatMessage({ id, params }) {
+                return intl.formatMessage({ id }, params);
+            },
+        },
+    };
+
+    return <AgChartsReact options={options} />;
+}
+
+const App = () => {
+    return (
+        <IntlProvider locale="fr-FR" messages={messages}>
+            <MyChart />
+        <IntlProvider>
+    );
+}
+```
+
+{% /tabItem %}
+
+{% tabItem id="React I18Next" label="React I18Next" %}
+
+```js
+import i18next from 'i18next';
+import { initReactI18next, useTranslation } from 'react-i18next';
+
+const messages = {
+    'context-menu.download': 'Téléchargez une copie de ce tableau',
+};
+
+await i18next.use(initReactI18next).init({
+    lng: 'fr',
+    resources: {
+        fr: {
+            translation: messages,
+        },
+    },
+});
+
+const MyChart = () => {
+     const { t } = useTranslation();
+
+    const options = {
+        // ...
+        locale: {
+            messages: messages,
+            formatMessage({ id, params }) {
+                return t(id, params);
+            },
+        },
+    };
+
+    return <AgChartsReact options={options} />;
+}
+```
+
+{% /tabItem %}
+
+{% /if %}
+
+{% /tabs %}
+
+## All Built-In Translations
+
+```js
```

Comment:
Any way for us to just inline the `en.ts` file here, so we can reduce maintenance overhead and potential for drift?

Reply:
Updated